### ISSUE TITLE
checkpoint inference

### ISSUE THREAD

**Issue by VafaKnm** [ylacombe/finetune-hf-vits#20]
Hi!
I have set the parameters so that the model does not push to huggingface and the checkpoints are saved directly in my local system. Each checkpoint folder has these files:
![ttttt](https://github.com/ylacombe/finetune-hf-vits/assets/103993288/7ea0545b-5a9b-4581-88a4-8a1be42a0693)

As you see, there are two optimizer, pytorch_model and scheduler. Is this normal? if so, how i can use it?

**Comment by VafaKnm** [comment]
Oh, sorry i found the answer.

**Comment by HesamAlavian** [comment]
@VafaKnm What is the answer? 
I have same problem.

**Comment by FacundoMartinezCampos** [comment]
How did you combined the model and the discriminator into the safetensor file?

**Comment by MendoLeo** [comment]
HI @VafaKnm , have the same problem  of your opened and closed issues, how to solve it please.
